doc/whats_new.rst

```diff
@@ -123,6 +123,14 @@ Changes
   `#2821 <https://github.com/nilearn/nilearn/pull/2821>`_ in order to match BIDS
   semantics. The use of `sessions` in :func:`nilearn.image.clean_img` is deprecated and
   will be removed in 0.10.0.
+- Display objects have been reorganized. For example, Slicers (like the
```

Comment:
maybe we should make these new modules start with `_` at least for now? that gives us the freedom to do some refactoring later if we want. also from the users' point of view this reorganization shouldn't matter because they can still import everything from `displays` as they used to.